pytripgui/canvas_vc/canvas_controller.py

```diff
@@ -90,10 +92,38 @@ def set_patient(self, patient, state):
             self._ui.voi_list.event_callback = self._on_update_voi
             self._ui.voi_list.fill(patient.vdx.vois, lambda item: item.name)
 
-        self._ui.set_position_changed_callback(self.set_current_slice_no)
         self.update_canvas_view()
+
+        self._update_slider_values(position_safe=True, perspective_safe=True)
+
         self._ui.draw()
 
+    def _update_slider_values(self, position_safe: bool, perspective_safe: bool):
+        """
+        Updates min, max and current values of ui slider.
+        Can be "safe" position-callback-wise and perspective-callback-wise,
+            which means that callback will not be invoked id proper flag is set
```

Comment:
A typo: `id` -> `if`

Can you also add some info what is the purpose of position change callback and perspective change callback ?

Reply:
Well, those are callback that existed already, but I will do it :)

Reply:
![image](https://user-images.githubusercontent.com/44035398/131684365-35115e4f-280f-42fe-8d66-58c6065849ec.png)